src/SumOfAllNumbersInString.java

```diff
@@ -0,0 +1,16 @@
+class SumOfAllNumbersInString {
+	void findSumOfAllNumbers(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index))) {
+				sum = sum + Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));